specification/automanage/resource-manager/Microsoft.Automanage/preview/2021-04-30-preview/automanage.json

```diff
@@ -1394,7 +1394,8 @@
       "type": "array",
```

Comment:
We are using arrays in 5 more places. Do we need to add x-ms-identifiers for other arrays in line @@1294, @@1307, @@1421,  @@1498, @@1566?

Reply:
And only added xms ids where it is required - this command will target the files specificied under the most recent tag in readme